.env

```diff
@@ -15,7 +15,7 @@ _APP_SYSTEM_TEAM_EMAIL=team@appwrite.io
 _APP_EMAIL_SECURITY=security@appwrite.io
 _APP_EMAIL_CERTIFICATES=certificates@appwrite.io
 _APP_SYSTEM_RESPONSE_FORMAT=
-_APP_OPTIONS_ABUSE=disabled
```

Comment:
I think we can keep abuse disabled during development.

On the other hand, if we want it enabled, it would be cool to also add tests now that abuse is enabled during CI/CD